os_net_config/impl_nmstate.py

```diff
@@ -75,8 +75,8 @@
     else
         driver="$def_driver"
     fi &&
-    cur_drv=$(readlink "/sys/bus/pci/devices/$vf_pci_id/driver") &&
-    cur_drv=$(basename "$cur_drv") &&
+    cur_drv=$(readlink "/sys/bus/pci/devices/$vf_pci_id/driver" 2>/dev/null) &&
+    cur_drv=$(basename "$cur_drv")
     if ! [ "$cur_drv" = "$driver" ]; then
```

Comment:
Isn't the logic here changed with the removal of the && in L79, so now L80 is always interpreted and $cur_drv is always compared to $driver? But isn't it possible that $cur_driver has no value? This isn't a problem, is it?